src/cdk-experimental/drag-drop/drag.ts

```diff
@@ -249,7 +250,7 @@ export class CdkDrag<T = any> implements OnDestroy {
       this._updateActiveDropContainer(event);
     } else {
       const activeTransform = this._activeTransform;
-      const {x: pageX, y: pageY} = this._getPointerPositionOnPage(event);
+      const {x: pageX, y: pageY} = this._getLockedPointerPositionOnPage(event);
```

Comment:
`_getLockedPointerPositionOnPage` -> `_getConstrainedPointerPosition`, to capture cases where it might be contained by something else (e.g. viewport boundaries)?